lighthouse-core/test/lib/page-functions-test.js

```diff
@@ -177,16 +179,19 @@ describe('Page Functions', () => {
       assert.equal(pageFunctions.getNodeLabel(el), Array(78).fill('a').join('') + '💡…');
     });
 
-    it('Uses tag name for html tags', () => {
-      const el = dom.createElement('html');
-      assert.equal(pageFunctions.getNodeLabel(el), 'html');
+    it('Returns selector as fallback if nodeLabel equals html tag name', () => {
```

Comment:
let's move this test to it's own `describe` block since it tests `getNodeDetails` instead of `getNodeLabel`

Reply:
will do!